scripts/time-filter.js

```diff
@@ -0,0 +1,118 @@
+import { textSummary } from 'https://jslib.k6.io/k6-summary/0.0.3/index.js';
+import http from 'k6/http';
+import {
+    check,
+    sleep
+} from 'k6';
+import {
+    generateRandomCoordinate,
+    getCapitalCoordinates,
+    destinations,
+    timeFilterOptions,
+    setThresholdsForScenarios
+} from './common.js';
+
+export let options = timeFilterOptions
+
+setThresholdsForScenarios(options)
+
+export default function() {
+    const appId = __ENV.APP_ID;
+    const apiKey = __ENV.API_KEY;
+    const host = __ENV.HOST || 'api-dev.traveltimeapp.com';
+    const country = __ENV.COUNTRY || 'GB'
+    const countryCode = country.toUpperCase()
+    const url = `https://${host}/v4/time-filter`;
+    const transportation = __ENV.TRANSPORTATION || 'driving+ferry'
+    const travelTime = __ENV.TRAVEL_TIME || 1900
+    const destinationsAmount = __ENV.SCENARIO_DESTINATIONS || 2000
+    const rangeWidth = __ENV.RANGE || 0
+    const rangeSettings = {
```

Comment:
rangeSettings should be disabled by default

Reply:
yes, range is not defined, it is 0. If range is 0, I disable it: `enabled: rangeWidth === 0 ? false : true`

Reply:
I see, then it's fine